content/faq/regtest-setup-guide.md

```diff
@@ -3,156 +3,188 @@ title: How do I setup an isolated regtest network to test LBRY?
 category: developer
 ---
 
-## Prerequisites
+## Why use a regtest server
+A regtest server provides for a way to instantly generate blocks so that transactions can be instantaneous, so ultimately no waiting for confirmations from the blockchain. Also, no problem if you accidently corrupt your wallet, since no real funds are lost! Just delete the files and setup a new one.
+
+## Setup
 
 To begin setting up the network, there are a few things you need.
 
-Note that you need a Linux distribution to run all this. A virtual machine is
-fine.
+Note that you need a Linux distribution to run all this. A virtual machine is fine.
 
-### virtual environment
+### Virtual Environment
 
-First up it's a good idea to create a Python virtual environment. This requires
-you to have a functional python2.7 setup, with the Python package manager `pip`
-installed. To create a new virtual environment in a folder `lbry-env`, run this:
-`virtualenv -p /usr/bin/python2.7 lbry-env` To enter the environment, run
+First up it's a good idea to create a Python virtual environment. This requires you to have a functional python2.7 setup, with the Python package manager `pip` installed. To create a new virtual environment in a folder `lbry-env`, run this:
+`virtualenv -p /usr/bin/python2.7 lbry-env`
+To enter the environment, run:
 `source lbry-env/bin/activate`.
 
 ### lbrycrd
 
-You need to download a build of `lbrycrd` from [here](https://github.com/lbryio/lbrycrd/releases/), no installation required.
-To configure `lbrycrd` you to create a file at `~/.lbrycrd/lbrycrd.conf`,
+You need to download a build of `lbrycrd` from [here](https://github.com/lbryio/lbrycrd/releases/), no installation required. To configure `lbrycrd` you need to create a file at `~/.lbrycrd/lbrycrd.conf`,
 containing the following.
 ```ini
 rpcuser=test
 rpcpassword=test
 rpcport=18332
-rpcallowip=0.0.0.0/0
 regtest=1
-dnsseed=0
-upnp=0
 server=1
-txindex=0
-daemon=0
+txindex=1
+daemon=1
+listen=0
+discover=0
 ```
 
-### lbry
+### lbryum-server
 
-Download source from [here](https://github.com/lbryio/lbry/releases), and run the following inside the environment.
+To install lbryum-server you first need to install the package `leveldb`. After that, download the source from [here](https://github.com/lbryio/lbryum-server/releases), and run the following _not_ inside the environment.
 ```bash
-cd lbry
-pip2 install -r requirements.txt
-pip2 install .
+cd lbryum-server
+sudo pip2 install -r requirements.txt
+```
+
+If you're not running debian/*buntu, or a derivative of those, you need to edit the `configure` file a bit. On line 11, remove the `apt-get` line and manually install the required packages. On line 51, change `adduser` to `useradd` and on the same line, change `--disabled-password` to `-p !`.
+```bash
```

Comment:
I assume this is instructions for mac OS? Maybe we can just add this changed file as a new file in the lbryum-server repo?

Reply:
I don't have it available, it was already in the previous version of the guide so I let it remain there. But it seems these are the instructions for running it on ArchLinux and similar OSes.